printer/opprec.go

```diff
@@ -0,0 +1,78 @@
+package printer
+
+import (
+	"fmt"
+
+	"github.com/haya14busa/go-vimlparser/ast"
+	"github.com/haya14busa/go-vimlparser/token"
+)
+
+const (
+	lowestPrec  = 0 // non-operators
+	highestPrec = 7
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
const highestPrec is unused (U1000)